### PR TITLE
DOC: Change mielke distribution default parameters to ensure kurtosis exists.

### LINKED ISSUE #9833: IntegrationWarning from mielke.stats() during build of html doc.
An IntegrationWarning occurs while building the docs.

### Reproducing code example:
```
$ pushed doc && make clean &&  make html
```

### Error message:
```
.../scipy/scipy/stats/_distn_infrastructure.py:1613: IntegrationWarning: The integral is probably divergent, or slowly convergent.                                                      
  return integrate.quad(self._mom_integ1, 0, 1, args=(m,)+args)[0]
```

After some digging, the culprit is the `mielke` distribution.  The doc creation calls
```
mean, var, skew, kurt = mielke.stats(k, s, moments='mvsk')
```

The `mielke`'s default parameters are `(k, s)= (10.4, 3.6)`.  The kurtosis calculation does not converge if `s <= 4`, and a warning is generated.

### Scipy/Numpy/Python version information:
```
1.3.0.dev0+12175d2 1.16.0 sys.version_info(major=3, minor=6, micro=6, releaselevel='final', serial=0)
```

### PR DESCRIPTION
**Pull request by pvanmulbregt**
The kurtosis calculation for mielke distribution (parameters `k` & `s`) does
not converge if `s` is too small.  (See gh-9833)  Change the default parameters for testing from `(10.4, 3.6)` to `(10.4, 4.6)`. (Any value of `s` bigger than `4` should be sufficient to ensure mathematical convergence. `4.6` is big enough that there are no computational convergence problems either.) 
This removes a warning during building of the doc, which calls `mielke.stats(k, s, moments='mvsk')`

Also added a reference for the distribution.

Closes gh-9833.

### CODE REVIEW

**Comment by pvanmulbregt** [comment]
The single CI failure is a pypy3 failure in `scipy/io/matlab/tests/test_streams.py::TestZlibInputStream::test_seek`.  

```
[gw0] [ 10%] PASSED scipy/io/matlab/tests/test_streams.py::TestZlibInputStream::test_read 
scipy/io/matlab/tests/test_streams.py::TestZlibInputStream::test_seek 
[gw0] node down: Not properly terminated
[gw0] [ 10%] FAILED scipy/io/matlab/tests/test_streams.py::TestZlibInputStream::test_seek 
Replacing crashed worker gw0
```

The `test_read()` method passes, but `test_seek()` does not. The failure doesn't appear to be related to this PR.  Is this test problematic in pypy3?

File: scipy/stats/_continuous_distns.py [hunk 1]
```diff
@@ -4583,7 +4583,7 @@ def _entropy(self):
 
 
 class mielke_gen(rv_continuous):
-    r"""A Mielke's Beta-Kappa continuous random variable.
+    r"""Mielke's Beta-Kappa continuous random variable[1]_.
```
Comment (chrisb83) [inline scipy/stats/_continuous_distns.py]: this text shows up in the overview here: https://docs.scipy.org/doc/scipy/reference/stats.html

Better put the reference to the notes

**Review by chrisb83 — COMMENTED** [review]
Good idea to change the parameters, just a minor comment where to cite [1]

**Comment by chrisb83** [comment]
Looks good, all tests passed.

**Pull request merged**